authorization_test.go

```diff
@@ -16,6 +16,7 @@ const (
 const (
 	validEmptyHeaderRsa          = "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.e30.YbkSeO9GIBYedph1uSQz0Y6zp1NwDEB8O7ek3cc3Vw4Fjh6DwrJAwmXoNSqT6FhHDv14QG70qPIuyzsR0Q9nHFo7hGEqE8E85F8z3Pj5eBjHKBMJFno7jww514Vyp35c490ZHD6_d3F9PmxWrPkKezc1mcwlCegwiMJIS2CeR7k"
 	validFullHeaderRsa           = "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJtZXJjdXJlIjp7InB1Ymxpc2giOlsiZm9vIiwiYmFyIl0sInN1YnNjcmliZSI6WyJmb28iLCJiYXoiXX19.B-ZBdmCbtDaP27wB_DvF9xIetQm88M2Q1d-LP2DZoEHrz6lYDuHkgXzSDnFdbLCZ653e0r_VOaKxe2Pc6R4F0ok2vksC6P5gHhqIUcQuTSlzNFyTrg4tyy4mMkcm1h85te9gkV4LR6TABfZpFPqqIS4t7fpCMxvtAkyf_RR5Fq4"
+	validFullHeaderRsaForCert    = "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IkxIbzlPMmNNUzBqbzRsQWwtRk11ayJ9.eyJodHRwczovL21lcmN1cmUucm9ja3MvIjp7InB1Ymxpc2giOlsiZm9vIiwiYmFyIl0sInN1YnNjcmliZSI6WyJmb28iLCJiYXoiXX0sImlzcyI6Imh0dHBzOi8vbWVyY3VyZS10ZXN0LmV1LmF1dGgwLmNvbS8iLCJzdWIiOiJhdXRoMHw1ZmRjM2U4OGUzYjA0YjAwNzZhNTQxM2MiLCJhdWQiOlsiaHR0cHM6Ly9tZXJjdXJlLnJvY2tzL2FwaSIsImh0dHBzOi8vbWVyY3VyZS10ZXN0LmV1LmF1dGgwLmNvbS91c2VyaW5mbyJdLCJpYXQiOjE2MDgyNzAxMTQsImV4cCI6MTYwODM1NjUxNCwiYXpwIjoiMzAxaHptQkEydnlnN2dKVmJIRUxSVEN6WXR1QmtVTnYiLCJzY29wZSI6Im9wZW5pZCBwcm9maWxlIGVtYWlsIn0.WhMkaOvIckY7PFCYs5SvIcl8OK32z7AhCCPGx0G3yF4L0nOTssXV9gAPEpEOrLbCOG3ALDxOGB4VagGnwBIYuztBsuZyPRoIwUkBZOgrIUJYS96jYTb9osPUYZ7BxNlVefFse93JmeSFTZRi6oH5lbqCEW6FUVKNlHWBBl39UK9Fg36EFtOHIJ7wZ_NX51TPvN-roCp27qIhY3atDcHYWXTKS7VjznKDKxl5G7AmyA1L7eE3vpnGiECFcLwxr9BJGVQmnPnwUKf-tY3pSBY0gFE562y15vXk3D2il43uAP4uK2sw8rcup_CmWnT3wmAdIODkwdn8zKM3GC3Y-9WEvQ"
```

Comment:
This token contains an `exp` claim and has expired, this breaks the tests. Could you provide a token without this claim please @TamasSzigeti?

Reply:
I was going to but I see you already tackled it (: Thank you

Reply:
Fixed by #428, don't bother @TamasSzigeti!